environment.yml

```diff
@@ -4,62 +4,75 @@ channels:
   - defaults
 
 dependencies:
-  - armadillo==8.200.0
-  - blas==1.1
-  - bullet==2.86.1
-  - bz2file==0.98
-  - bzip2==1.0.6
-  - cmake>=3.10
-  - cspice==66
-  - curl==7.60.0
-  - doxygen==1.8.14
-  - eigen==3.3.3
-  - embree==2.14.0
-  - geos==3.5.1
-  - geotiff==1.4.2
-  - gmm==5.0
-  - gmp==6.1.2
-  - gsl==2.2.1
-  - hdf5==1.8.18
-  - icu==58.2
-  - jama==125
+  - armadillo
+  - blas
+  - bullet
+  - bz2file
+  - bzip2
+  - cspice
+  - curl
+  - doxygen
+  - eigen
+  - embree
+  - geos
+  - geotiff
+  - gmm
+  - gmp
+  - gsl
+  - hdf5
+  - icu
+  - jama
   - jpeg==9b
   - kakadu==1
-  - krb5==1.14.2
+  - krb5
   - libpng>=1.6.34
-  - libprotobuf==3.5.2
-  - libtiff>=4.0.9
-  - libxml2==2.9.7
+  - libprotobuf
+  - libtiff
+  - libxml2
   - make
-  - mesalib==17.2.0
-  - mysql==5.7.20
-  - mysql-connector-c==6.1.6
-  - nanoflann==1.2.2
-  - ninja==1.7.2
+  - mesalib
+  - mysql
+  - mysql-connector-c
+  - nanoflann
+  - ninja
   - conda-forge/label/gcc7::nn
-  - openblas==0.2.19
-  - opencv
-  - openssl==1.0.2n
-  - pcl==1.8.1
-  - pip==9.0.1
-  - protobuf==3.5.2
-  - python==3.6
-  - qhull==7.2.0=0
+  - opencv=3.3.0
+  - openssl
+  - pcl
+  - pip
+  - protobuf>=3.5.2
+  - python
```

Comment:
Removed python since it's not a direct requirement of ISIS, was originally from the SIP stuff but I imagine we are not going through with that anymore.